docs/concepts.md

```diff
@@ -0,0 +1,55 @@
+concepts:
+
+```
+grip
+reps
+front types
```

Comment:
It would be nice for the `src/protocol` docs to be separate because that directory will be a standalone package that other projects will import. It is also helpful to differentiate between devtools, src/protocol, and https://www.npmjs.com/package/@recordreplay/protocol

Reply:
https://replay.io/protocol